bucket/git-credential-manager-core.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.475",
+    "description": "Git Credential Manager Core (GCM Core) is a secure Git credential helper built on .NET that runs on Windows and macOS. Linux support is in an early preview.",
```

```suggestion
    "description": "Secure Git credential helper"
```
We tend to keep the description shorter, so that we can fit more information in a short paragraph.